### ISSUE TITLE
Pre-built Support for Ruby 3.2 (or 3.3)?

### ISSUE THREAD

**Issue by eric-hemasystems** [GoogleCloudPlatform/ruby-docker#231]
Looks like the [pre-buily Ruby binaries](https://github.com/GoogleCloudPlatform/ruby-docker/blob/master/ruby-pipeline/ruby-latest.yaml) stopped at 3.1.2 while the 3.2 series came out about 7-8 months ago. Any way GAE can be updated to support the latest versions of Ruby?

**Comment by eric-hemasystems** [comment]
I guess now that Ruby 3.3 is out it would be nice to be able to use that without GAE having to compile the runtime with every release.

**Comment by eric-hemasystems** [comment]
Weirdly I'm now also seeing this when I deploy to GAE:

```
WARNING: ruby 3.1 and earlier versions will reach end of support on 2024-07-10 for App Engine flexible environment. After 2024-07-10, you cannot deploy new or re-deploy existing applications that use runtimes after their end of support date. We recommend that you migrate to the latest supported version of ruby.
```

This despite the fact that I'm using Ruby 3.3 (so maybe it unconditionally prints this message?).

Seemed odd to me that you were dropping support for the latest version that you have pre-compiled binaries for. Also even if I was using Ruby 3.1, 2024-07-10 is well before [EOL of that branch](https://www.ruby-lang.org/en/downloads/branches/). Any specific reason that GAE is dropping support before EOL? Is that something that will happen in future versions of Ruby?

**Comment by eric-hemasystems** [comment]
Also for older Ruby versions the latest patch release is not available. For example, 3.0.6 is the latest of the 3.0.x series but prebuilt binaries are only available up to 3.0.4 missing out on two important CVEs.

**Comment by eric-hemasystems** [comment]
Closing now this this repo is legacy. See my comment [here](https://github.com/GoogleCloudPlatform/ruby-docker/issues/240#issuecomment-2152778262) for more info.
